### PR TITLE
AMP-83919 [BigQueryGA4Import] Add notes of BigQuery import to all places mentioning GA4

### PR DESCRIPTION
**Pull request by bh202205**
# Amplitude Developer Docs PR

## Description

This PR adds notes of BigQuery import to all places mentioning GA4 to track customers/users attention for our support of GA4 import via BigQuery import.

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [x] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [x] My documentation follows the style guidelines of this project.
- [x] I previewed my documentation on a local server using `mkdocs serve`.
- [x] Running `mkdocs serve` didn't generate any failures.
- [x] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

**Review by joez2 — APPROVED** [review]
Previewed the change. LGTM. Thanks for updating the dev docs!

File: docs/data/destinations/google-analytics-4-firebase.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ Amplitude CDP's Google Analytics 4 (iOS/Android) streaming integration enables y
 
     The Google Analytics 4 (iOS/Android) destination works with an iOS or Android mobile application using Firebase. If you are working with a web application instrumented with Google Tag (gtag.js), set up a [Google Analytics 4 (Web)](../google-analytics-4-gtag) destination instead.
 
+!!!note "BigQuery Import for GA4 (Google Analytics 4) Beta"
+
+    We are working on BigQuery Import for GA4 Beta. Please feel free to reach out to <dwh+GA4beta@amplitude.com>. In addition, click [here](/data/sources/bigquery) to view how you can import BigQuery data into Amplitude.
```
Comment (markzegarelli) [inline docs/data/destinations/google-analytics-4-firebase.md:15]:
```suggestion
    Amplitude is working on BigQuery Import for GA4 Beta. Contact [dwh+GA4beta@amplitude.com](mailto:dwh+GA4beta@amplitude.com) to learn more. For more information about importing BiqQuery data in to Amplitude, see the [BigQuery Source documentation](/data/sources/bigquery).
```

File: docs/data/destinations/google-analytics-4-gtag.md [hunk 2]
```diff
@@ -9,6 +9,10 @@ Amplitude CDP's Google Analytics 4 (Web) streaming integration enables you to fo
 
       Google Analytics 4 (Web) destination works with a web application instrumented with Google Tag (gtag.js). If you are working with an iOS or Android mobile application using Firebase, set up a [Google Analytics 4 (iOS/Android)](../google-analytics-4-firebase) destination instead.
 
+!!!note "BigQuery Import for GA4 (Google Analytics 4) Beta"
+
+    We are working on BigQuery Import for GA4 Beta. Please feel free to reach out to <dwh+GA4beta@amplitude.com>. In addition, click [here](/data/sources/bigquery) to view how you can import BigQuery data into Amplitude.
```
Comment (markzegarelli) [inline docs/data/destinations/google-analytics-4-gtag.md:14]:
```suggestion
    Amplitude is working on BigQuery Import for GA4 Beta. Contact [dwh+GA4beta@amplitude.com](mailto:dwh+GA4beta@amplitude.com) to learn more. For more information about importing BiqQuery data in to Amplitude, see the [BigQuery Source documentation](/data/sources/bigquery).
```

File: docs/data/tutorials/migrating-from-google-analytics/index.md [hunk 3]
```diff
@@ -11,6 +11,10 @@ Migrating from the Google Analytics (GA4) SDK to the Amplitude SDK involves adju
 
     While completely replacing Google Analytics (GA4) tracking functions with Amplitude tracking functions is our preferred long-term solution for migrating, Amplitude recognizes that this process can be complex. Amplitude offers a [Google Analytics event forwarder](#using-google-analytics-events-forwarder) that can get you started with your migration quickly, with only one small code change.
 
+!!!note "BigQuery Import for GA4 (Google Analytics 4) Beta"
+
+    We are working on BigQuery Import for GA4 Beta. Please feel free to reach out to <dwh+GA4beta@amplitude.com>. In addition, click [here](/data/sources/bigquery) to view how you can import BigQuery data into Amplitude.
```
Comment (markzegarelli) [inline docs/data/tutorials/migrating-from-google-analytics/index.md:16]:
```suggestion
    Amplitude is working on BigQuery Import for GA4 Beta. Contact [dwh+GA4beta@amplitude.com](mailto:dwh+GA4beta@amplitude.com) to learn more. For more information about importing BiqQuery data in to Amplitude, see the [BigQuery Source documentation](/data/sources/bigquery).
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]
Thanks @bh202205!

I edited the content to align with our style elsewhere.

One question I have is, "BigQuery Import for GA4 Beta"...does this mean that we are working on a beta of the BigQuery Import feature? Or will the BigQuery Import feature work on a beta version of GA4?

**Comment by bh202205** [comment]
> Thanks @bh202205!
> 
> I edited the content to align with our style elsewhere.
> 
> One question I have is, "BigQuery Import for GA4 Beta"...does this mean that we are working on a beta of the BigQuery Import feature? Or will the BigQuery Import feature work on a beta version of GA4?

@markzegarelli Thank you for all your comments and just updated the files! 

To answer your question, let me give you some context. BigQuery is one of the popular data warehouses that our customers use and it's a google cloud product. We have a data warehouse integration called: BigQuery Import, which allow customers to import data from their BigQuery. GA4 (google analytics 4) is one of our competitors and what we are building is an kind of new integration that customers to import their GA4 data to Amplitude based on our existing BigQuery Import.

Let me know if this makes sense to you. Thank again!

File: docs/data/sources/bigquery.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ With Amplitude's BigQuery integration, you can ingest BigQuery data directly int
 
 !!!note "BigQuery Import for GA4 (Google Analytics 4) Beta"
 
-    We are working on BigQuery Import for GA4 Beta. Please feel free to reach out to <dwh+GA4beta@amplitude.com>.
+    Amplitude is working on BigQuery Import for GA4 Beta. Contact [dwh+GA4beta@amplitude.com](mailto:dwh+GA4beta@amplitude.com) to learn more. For more information about importing BiqQuery data in to Amplitude, see the [BigQuery Source documentation](/data/sources/bigquery).
```
Comment (markzegarelli) [inline docs/data/sources/bigquery.md:18]:
```suggestion
    Amplitude is working on a beta version of BigQuery Import for GA4. Contact [dwh+GA4beta@amplitude.com](mailto:dwh+GA4beta@amplitude.com) to learn more. For more information about importing BiqQuery data in to Amplitude, see the [BigQuery Source documentation](/data/sources/bigquery).
```

File: docs/data/tutorials/migrating-from-google-analytics/index.md [hunk 5]
```diff
@@ -11,6 +11,10 @@ Migrating from the Google Analytics (GA4) SDK to the Amplitude SDK involves adju
 
     While completely replacing Google Analytics (GA4) tracking functions with Amplitude tracking functions is our preferred long-term solution for migrating, Amplitude recognizes that this process can be complex. Amplitude offers a [Google Analytics event forwarder](#using-google-analytics-events-forwarder) that can get you started with your migration quickly, with only one small code change.
 
+!!!note "BigQuery Import for GA4 (Google Analytics 4) Beta"
+
+    Amplitude is working on BigQuery Import for GA4 Beta. Contact [dwh+GA4beta@amplitude.com](mailto:dwh+GA4beta@amplitude.com) to learn more. For more information about importing BiqQuery data in to Amplitude, see the [BigQuery Source documentation](/data/sources/bigquery).
```
Comment (markzegarelli) [inline docs/data/tutorials/migrating-from-google-analytics/index.md:16]:
```suggestion
    Amplitude is working on a beta version of BigQuery Import for GA4. Contact [dwh+GA4beta@amplitude.com](mailto:dwh+GA4beta@amplitude.com) to learn more. For more information about importing BiqQuery data in to Amplitude, see the [BigQuery Source documentation](/data/sources/bigquery).
```

**Pull request merged**